R/Registry.R

```diff
@@ -138,6 +138,29 @@ Registry <- R6::R6Class( # nolint object_name_linter
         # Bind entries as rows
         list_rbind()
     },
+    from_df = function(dataframe, key = NULL, description = NULL, run = NULL) {
+      if (private$.registry_name != "artifact") {
+        cli::cli_abort(
+          "Creating records from data frames is only supported for the Artifact registry"
+        )
+      }
+
+      check_requires("Creating records from data frames", "reticulate")
+
+      py_lamin <- reticulate::import("lamindb")
```

Comment:
Should we also use `check_requires` for python packages?

```suggestion
      check_requires("Creating records from data frames", "reticulate")
      check_requires("Creating records from data frames", "lamindb", "python")

      py_lamin <- reticulate::import("lamindb")
```

Reply:
Or

```suggestion
      check_requires("Creating records from data frames", "reticulate")
      check_requires("Creating records from data frames", "lamindb", "reticulate")
      py_lamin <- reticulate::import("lamindb")
```

Reply:
Possibly. I was going to change how/when we check for Python **lamin** anyway.